build_v3.d

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env rdmd
+/**
+Phobos V3 Build Script
+
+Usage:
+  ./build_v3.d [debug,release,unittest]
+*/
+
+import std.conv;
+import std.datetime;
+import std.file;
+import std.path;
+import std.process;
+import std.stdio;
+import std.string;
+
+int main(string[] args)
+{
+    int result = 0;
+    
+    bool buildUnittest = false;
+    bool buildRelease = false;
+    if (args.length > 1)
+    {
+        buildUnittest = args[1] == "unittest";
+        buildRelease = args[1] == "release";
```

Comment:
This is fine for the moment, but we are going to want to change it so that the normal unittest build does both debug and release builds (which is what `make unittest` does) - as well as probably having a way to indicate that you want to do a unittest build that's only debug or only release. That's currently handled with `BUILD=debug` or `BUILD=release`.

But I think that we can make those improvements with a separate PR.